ios/Video/RCTVideo.swift

```diff
@@ -599,8 +601,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         _localSourceEncryptionKeyScheme = keyScheme
     }
 
-    func playerItemPrepareText(source: VideoSource, asset: AVAsset!, assetOptions: NSDictionary?, uri: String) async -> AVPlayerItem {
-        if source.textTracks.isEmpty != true || uri.hasSuffix(".m3u8") {
+    func playerItemPrepareText(asset: AVAsset!, assetOptions: NSDictionary?, uri: String) async -> AVPlayerItem {
+        if self._textTracks.isEmpty == true || (uri.hasSuffix(".m3u8")) {
```

Comment:
The logic change here ?

Reply:
Whops, will fix rebase!

Reply:
On master we have `source.textTracks.isEmpty != true` - this is expected ?

Reply:
I think you are right in fact, give me 10 min to double check

Reply:
@KrzysztofMoch OK, You are fixing a regression I introduced ...

Reply:
it should be fine now